paasta_tools/oom_logger.py

```diff
@@ -184,6 +189,8 @@ def main():
         env_vars = get_container_env_as_dict(docker_inspect)
         service = env_vars.get("PAASTA_SERVICE", "unknown")
         instance = env_vars.get("PAASTA_INSTANCE", "unknown")
+        mesos_container_id = env_vars.get("MESOS_CONTAINER_NAME", "mesos-null")
```

Comment:
Smart, then we can use the same regex and get "null" as the container id for k8s tasks :) Took me a minute to figure out why we were doing this.

Reply:
yeah a bit hacky :-D